packages/admin/admin-rte/src/core/extension/Link/ToolbarButton.tsx

```diff
@@ -120,14 +121,14 @@ function LinkDialog(props: {
                     <Grid container spacing={4}>
                         {linkData && (
                             <Grid item>
-                                <Button variant="contained" startIcon={<Delete />} onClick={handleRemove}>
+                                <Button startIcon={<Delete />} onClick={handleRemove}>
```

Comment:
Although the design wasn't correct previously, this still changes the design, as `primary` is the default variant of the Comet Button. 

See the Link Dialog in the RTE of this story: https://storybook.comet-dxp.com/?path=/story/comet-admin-rte--rte-all-options

```suggestion
                                <Button variant="outlined" startIcon={<Delete />} onClick={handleRemove}>
```

| Previously | Now |
| --- | --- |
| <img width="692" alt="Screenshot 2025-03-03 at 11 13 04" src="https://github.com/user-attachments/assets/b99b7d81-53f4-42d0-a650-c17717588771" /> | <img width="692" alt="Screenshot 2025-03-03 at 11 13 22" src="https://github.com/user-attachments/assets/03019232-8591-4347-b93c-182fd183c6ee" /> |